### PR TITLE
dotnetcli:chore - improve tests asserts and code cleaning

### PR DESCRIPTION
**Pull request by matheusalcantarazup**
Previously the mock output from success tests was a bit hard to read and
understand, and we was not asserting that the formatter was parsing
correctly the output from dotnet cli.

Since the formatter needs a .csproj file to check the dependencies the
function created on #806 to copy files from examples directory to a
.horusec directory was moved to testutil package (and changed to be more
generic) and the references on nancy tests was updated.

This commit improve the tests to use .csproj files from examples
directory and new asserts was added to verify that the output was
parsed correctly.

The schema definition and enums was also moved from dotnet_cli/entities
and dotnet_cli/enums to dotnet_cli package and changed to be private.

Since the const `CsProjExt` is used on tests of `utils/file` package the
package name of these tests was renamed to `file_test` to avoid
cyclomatic imports.

Updates #718

Signed-off-by: Matheus Alcantara <matheus.alcantara@zup.com.br>

**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: internal/utils/testutil/utils.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
```
Comment (iancardosozup) [inline internal/utils/testutil/utils.go:1]: i think the filename `utils.go` is a bit generic but couldn't think about a assertive name

Comment (matheusalcantarazup) [reply internal/utils/testutil/utils.go:1]: I have the same felling...

Maybe generic functions that can't have a specific file name can leave on this generic file.  What do you think @nathanmartinszup ?

Comment (nathanmartinszup) [reply internal/utils/testutil/utils.go:1]: Maybe we can move the function to the path.go file or create a file for directory functions.

Comment (matheusalcantarazup) [reply internal/utils/testutil/utils.go:1]: I think that path would be good. I'll move.

**Review by nathanmartinszup — APPROVED** [review]

**Review by iancardosozup — APPROVED** [review]

**Pull request merged**